### PR TITLE
ResNet pretrained, Efficient training

### PR DESCRIPTION
**Pull request by Arkkienkeli**
Hi Juan, could you have a look at the code and provide feedback? 
I checked this code on a very small portion of TA-ORF (~10 images) and it worked fine. 
If everything is fine with the code, please don't merge it just yet.
I am going to run tests for the whole TA-ORF dataset and get back here with the results.

### CODE REVIEW

**Comment by Arkkienkeli** [comment]
Hi Juan, I checked on the full TA-ORF dataset and it works completely similarly. 
image

**Review by jccaicedo — COMMENTED** [review]
The updates to EfficientNet look good, except for the model type.
Can you briefly explain what were the changes made to ResNet? I thought we were adding the pre-trained capability, but it seems like the training mode has been updated instead, so I'm confused.

The results look great, though! So that is reassuring.

File: plugins/models/efficientnet.py [hunk 1]
```diff
@@ -26,14 +28,21 @@ def get_supported_models(self):
             7: efn.EfficientNetB7,
```
[the remark this answers is not in the document]
Comment (Arkkienkeli) [reply plugins/models/efficientnet.py:28]: We can choose with `conv_blocks : [0-7]`  param in the config.

**Review by Arkkienkeli — COMMENTED** [review]
I had to reorganize the code a bit so I could switch between pretrained mode or training mode.

File: plugins/models/efficientnet.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jccaicedo) [reply plugins/models/efficientnet.py:28]: Understood! This makes sense.

**Review by jccaicedo — APPROVED** [review]
OK. Ready to merge.
